lib/di/global/get_it_initializer.dart

```diff
@@ -268,5 +269,8 @@ class GetItInitializer {
     getIt.registerLazySingleton<UpdatePinnedMessagesInteractor>(
       () => UpdatePinnedMessagesInteractor(),
     );
+    getIt.registerFactory<SendFilesOnWebWithCaptionInteractor>(
+      () => SendFilesOnWebWithCaptionInteractor(),
```

Comment:
how about the problem which you said about the order of the message?

Reply:
i have tried both `registerFactory` and `registerSingleton`. But, the order of messages is not correct, so i think we can do it in another ticket